modules/images-managing-images-enabling-imagestreams-kube.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing-images/using-imagestreams-with-kube-resources.adoc
+
+
+[id="images-managing-images-enabling-imagestreams-kube_{context}"]
+= Enabling imagestreams with Kubernetes resources
+
+When using imagestreams with Kubernetes resources, you can only reference imagestreams that reside in the same project as the resource. The imagestream reference must consist of a single segment value, for example `ruby:2.5`, where `ruby` is the name of an imagestream that has a tag named `2.5` and resides in the same project as the resource making the reference.
+
+[NOTE]
+====
+This feature can not be used in the `default` namespace, nor in any `openshift-` or `kube-` namespace.
+====
+
+There are two ways to enable imagestreams with Kubernetes resources:
+
+* Enabling imagestream resolution on a specific resource. This allows only this resource to use the imagestream name in the image field.
+* Enabling imagestream resolution on an imagestream. This allows all resources pointing to this imagestream to use it in the image field.
+
+.Procedure
+
+. Use `oc set image-lookup` to enable imagestream resolution on a specific resource or imagestream resolution on an imagestream. For example,
+the following command allows all resources to reference the imagestream named
+`mysql`:
++
+[source,terminal]
+----
+$ oc set image-lookup mysql
+----
++
+This sets the `Imagestream.spec.lookupPolicy.local` field to true.
++
+.Imagestream with image lookup enabled
+[source,yaml]
+----
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/display-name: mysql
+  name: mysql
+  namespace: myproject
+spec:
+  lookupPolicy:
+    local: true
+----
++
+When enabled, the behavior is enabled for all tags within the imagestream.
++
+. Then you can query the imagestreams and see if the option is set:
++
+[source,terminal]
+----
+$ oc set image-lookup imagestream --list
+----
+
+////
```

Comment:
do these slashes mean this is commented out?

Reply:
(I wouldn't expect this section to be commented out which is why i ask)

Reply:
Yeah - they were there previously (and I honestly can't remember why), but if the content is valid I'll remove them.

Reply:
it's valid (the content shows up in the 3.11 docs)

Reply:
Thanks! Updated.